system/HTTP/RedirectException.php

```diff
@@ -0,0 +1,3 @@
+<?php namespace CodeIgniter\HTTP;
+
+class RedirectException extends \Exception {}
```

Comment:
Well, I guess you can always define a Common package and put there the core functions, custom exceptions, and so on. So our specific packages depend only on one small generic package Common, which is fine, I think.